tools/IceRpc.BuildTelemetry.Reporter/README.md

```diff
@@ -0,0 +1,35 @@
+# Build Telemetry
+
+`IceRpc.Protobuf.Tools` and `IceRpc.Slice.Tools` collect anonymous
+build telemetry data about general usage. Participation in this anonymous
+program is optional, and you may [opt-out](#opting-out) if you'd prefer to not
+share any information.
```

Comment:
Personally, I feel like we're overusing `anonymous` here. At best it's just redundant, at worst, stressing how anonymous it is makes it seem suspicious IMO.
```suggestion
`IceRpc.Protobuf.Tools` and `IceRpc.Slice.Tools` collect anonymous
build telemetry data about general usage. Participation in this program is
optional, and you may [opt-out](#opting-out) if you'd prefer to not share any 
information.
```